Cargo.toml

```diff
@@ -52,7 +52,8 @@ arbitrary = { version = "1", optional = true, default-features = false }
 ark-ff-03 = { version = "0.3.0", package = "ark-ff", optional = true, default-features = false }
 ark-ff-04 = { version = "0.4.0", package = "ark-ff", optional = true, default-features = false }
 bn-rs = { version = "0.2", optional = true, default-features = true }
-fastrlp = { version = "0.3", optional = true, default-features = false, features = [
+fastrlp-03 = { version = "0.3", package = "fastrlp", optional = true, default-features = false, features = ["alloc"] }
```

Comment:
nit: formatting should have new lines around `"alloc"` as on L57 (2 lines down)

Reply:
yep, done on https://github.com/recmo/uint/pull/403/commits/947d0f5bcc998f07e0c1bf15f193b31360fd2294